### DISCUSSION TITLE
ww3_shel run error for ww3 6.07.

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#919] [General]
I had compiled ww3 6.07 based what it says in the manual and when I ran ww3_prep I could the following:

./ww3_prep

                 *** WAVEWATCH III  Input pre-processing ***  
               ===============================================

 *** WAVEWATCH III ERROR IN W3PREP : 
     ERROR IN OPENING INPUT FILE
     IOSTAT =    2

which is to be expected when running ww3_prep without an input file.

However when I run ww3_shel I get the following output:

./ww3_shel

                     *** WAVEWATCH III Program shell ***      
               ===============================================

At line 919 of file ww3_shel.F90 (unit = 11, file = 'fort.11')
Fortran runtime error: End of file

Error termination. Backtrace:
#0  0x7efdb5ef1171 in ???
#1  0x7efdb5ef1d19 in ???
#2  0x7efdb5ef2521 in ???
#3  0x7efdb60fa1bb in ???
#4  0x7efdb60fa7b2 in ???
#5  0x7efdb60f733b in ???
#6  0x7efdb60fbec4 in ???
#7  0x7efdb60fcd34 in ???
#8  0x4024f8 in ???
#9  0x401eb6 in ???
#10  0x7efdb536bd84 in ???
#11  0x401eed in ???
#12  0xffffffffffffffff in ???

for wave watch 3 6.07.   How do I resolve this issue?  Was there a error that needs fixing in ww3_shel.F90?

**Comment by JessicaMeixner-NOAA** [comment]
Have you run ww3_grid?  That's the first step before running ww3_prep.

**Reply by JMSnyder1980** [reply]
Yes I did.  The point I am trying to get at is when I run ww3_grid or ww3_prep I just get a missing input error.  When I run ww3_shel I get a set fault type error.  Not sure why this is happening.  Attached are my input files for running ww3 v6.07

[ww3_grid.inp.txt](https://github.com/NOAA-EMC/WW3/files/10854469/ww3_grid.inp.txt)
[ww3_prep.inp.txt](https://github.com/NOAA-EMC/WW3/files/10854472/ww3_prep.inp.txt)
[ww3_shel.inp.txt](https://github.com/NOAA-EMC/WW3/files/10854473/ww3_shel.inp.txt)
